src/lib/utils/invalid_input_exception.hpp

```diff
@@ -0,0 +1,22 @@
+#pragma once
+
+#include <iostream>
+#include <exception>
+#include <stdexcept>
+#include <sstream>
+
+namespace opossum {
+
+
+/*
+ * Hyrise specific exception used to handle errors related to wrong user input.
+ * The console will catch this exception and (try to) rollback related transactions.
```

Comment:
I don't think we do a rollback in the sense of a mvcc-rollback, do we? if we don't then this comment would be confusing

Reply:
Good point. I misinterpreted the function call `_handle_rollback` in the console. It basically just checks if the transaction was aborted properly and not yet committed